pgml-extension/requirements.linux.txt

```diff
@@ -45,9 +53,12 @@ langchain-core==0.1.1
 langsmith==0.0.72
 lightgbm==4.1.0
 lxml==4.9.3
+markdown-it-py==3.0.0
 MarkupSafe==2.1.3
 marshmallow==3.20.1
 matplotlib==3.8.2
+maturin==1.4.0
```

Comment:
Don't think you need maturin inside PostgresML deployments. This may be a "leak" from the pypgrx extension venv.